tracer/build/_build/PrepareRelease/GenerateIntegrationDefinitions.cs

```diff
@@ -114,24 +114,27 @@ static void WriteCallTargetDefinitionFile(StreamWriter swriter, IEnumerable<Call
             swriter.WriteLine("// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.");
             swriter.WriteLine("// </copyright>");
             swriter.WriteLine();
+            swriter.WriteLine("using System.Collections.Generic;");
+            swriter.WriteLine();
             swriter.WriteLine("namespace Datadog.Trace.ClrProfiler");
             swriter.WriteLine("{");
             swriter.WriteLine("    internal static partial class InstrumentationDefinitions");
             swriter.WriteLine("    {");
 
             // Default Integrations
             var normalCallTargetIntegrations = callTargetIntegrations.Where(i => i.IntegrationType == IntegrationType.Default).Distinct().ToList();
-            swriter.WriteLine("        private static NativeCallTargetDefinition[] GetDefinitionsArray()");
+            swriter.WriteLine("        private static List<NativeCallTargetDefinition> GetDefinitionsList()");
             swriter.WriteLine("        {");
-            swriter.WriteLine("            return new NativeCallTargetDefinition[]");
-            swriter.WriteLine("            {");
+            swriter.WriteLine("            List<NativeCallTargetDefinition> definitionList = new List<NativeCallTargetDefinition>();");
```

Comment:
Separate note: if we know the number of items in advance, we should use the `new List<T>(int capacity)` constructor overload when initializing the list. This will reduce the number of times the list has to expand (which involves allocating new internal arrays and copying items over). It looks like we can use `normalCallTargetIntegrations.Count` for this.